docs/how-to/how-to-run-kata-containers-with-kinds-of-Block-Volumes.md

```diff
@@ -69,10 +69,83 @@ $sudo ctr run -t --rm --runtime io.containerd.kata.v2 --mount type=directvol,src
 ```
 
 
-### SPDK Device Based Volume
+### VFIO Device Based Volume
 
 TBD
 
-### VFIO Device Based Volume
+### SPDK Device Based Volume
+
+SPDK vhost-user devices in runtime-rs, unlike runtime (golang version), there is no need to `mknod` device node under `/dev/` any more.
+Just using the `kata-ctl direct-volume add ..` to make a mount info config is enough.
+
+#### Run SPDK vhost target and Expose vhost block device
+
+Run a SPDK vhost target and get vhost-user-blk controller as an example:
+
+First, run SPDK vhost target:
+
+> **Tips:** If driver `vfio-pci` supported, you can run SPDK with `DRIVER_OVERRIDE=vfio-pci`
+> Otherwise, Just run without it `sudo HUGEMEM=4096 ./scripts/setup.sh`.
+
+```bash
+$ cd /home/devel/spdk/
+$ VHU_UDS_PATH=/tmp/vhu-targets
+$ # Reset first
+$ ./scripts/setup.sh reset
```

Comment:
Suggest add a prefix like ${KATA_HOME}/scripts/..., so we will know where the setup.sh file is

Reply:
Thx.
Fixed as your suggestion.